netfile_client/NetFileClient.py

```diff
@@ -92,10 +93,12 @@ def get_auth(self, env_file):
 
     def fetch(self, endpoint, **kwargs):
         """ Fetch all of a particular record type """
+        self._logger.debug('Fetch got kwargs %s', kwargs)
         url = self._base_url + getattr(Routes, endpoint)
         params = { **self._params }
         if 'params' in kwargs:
             params.update(kwargs['params'])
+        self._logger.debug('Fetch %s with params %s', url, params)
```

Comment:
Didn't you say at some point that you preferred format strings?  I'm ok either way.

Reply:
This is a Pylint thing. Apparently "lazy-format" strings, i.e., %s-style, don't get interpolated unless that part of the code actually runs, unlike f-strings, .format(), etc. For logging, when a particular logger method might not run (like in this case, where if LOG_LEVEL=DEBUG is not set, this won't run), Pylint prefers lazy formatting.

Reply:
I'm using Pylint's default rules in the absence of anything else. I'm not wed to either Pylint or its default rules, but I think we should pick a linter and a set a rules for this repo. Then we can both run them in our editor and on CI.

Reply:
sounds good - I'll try setting up pylint going forward